api/app/Providers/AppServiceProvider.php

```diff
@@ -24,6 +24,10 @@ public function boot()
         // https://laravel.com/docs/9.x/eloquent#configuring-eloquent-strictness
         Model::shouldBeStrict(! $this->app->isProduction());
 
+        // https://github.com/PHPOffice/PHPWord/issues/2524#issuecomment-1847981808
+        // https://phpoffice.github.io/PHPWord/usage/introduction.html#output-escaping
+        \PhpOffice\PhpWord\Settings::setOutputEscapingEnabled(true);
```

Comment:
Odd that this is not the default :face_with_spiral_eyes:

Reply:
Yeah, for sure.  The explanation on the website is that they want to maintain backward comparability.  Just increment the major version number and fix it already!

Reply:
Reading through that page, we might want to update the paper to letter, as well.

https://phpoffice.github.io/PHPWord/usage/introduction.html#default-paper

Reply:
Must be an EU developed library :stuck_out_tongue: